### ISSUE TITLE
can we format minor labels

### ISSUE THREAD

**Issue by timelineAxis** [almende/vis#2464]
![image](https://cloud.githubusercontent.com/assets/24411932/21258728/506c4ad0-c3a4-11e6-9d36-8821f51c6c76.png)
I need 14 to be displayed below We and 15 below Th and so on

**Comment by yotamberk** [comment]
@timelineAxis you have a `format` option you can pass a function of the way you want to display the labels. It will look like this:
```javascript
function format({
  minorLabels: Function(date: Date, scale: Number, step: Number),
  majorLabels: Function(date: Date, scale: Number, step: Number)
}
```

**Comment by timelineAxis** [comment]
@yotamberk : I tried with  minorLabels: Function(date: Date, scale: Number, step: Number) {} but its not support "a \n b" or "a <br/> b"

**Comment by yotamberk** [comment]
@timelineAxis You right. It isn't possible yet. I've added a fix for this here: #2493

**Comment by yotamberk** [comment]
This will be available in the v4.18.0 or the current `develop` branch
